docs/excel/build-your-first-excel-add-in.md

```diff
@@ -10,7 +10,9 @@ You’ll start by creating a web app using HTML and JQuery. Then you'll create a
 
 ### Code it
 
-1- Create a folder on your local drive named QuarterlySalesReport (for example, C:\\QuarterlySalesReport). Save all files created in the steps that follow into this folder.
+#### Create the web app
```

Comment:
Do we need the Code it H2 heading here, or can we elevate the H3s to H2s? This removes the stacked heading. Also note that the H2 headings are what renders in the In this article control, so might be more relevant to have the specific tasks at the H2 level.